### ISSUE TITLE
Improve error messages on the failing Hypothesis deadline healthcheck

### ISSUE THREAD

**Issue by Stranger6667** [schemathesis/schemathesis#880]
The current version:

```
hypothesis.errors.DeadlineExceeded: Test took 748.20ms, which exceeds the deadline of 500.00ms

Add this option to your command line parameters to see full tracebacks: --show-errors-tracebacks
```

But actually, we can suggest to disable the deadline or put a different value there

**Comment by niecore** [comment]
What would be the correct configuration to disable the DeadlineExceeded check?

**Comment by Stranger6667** [comment]
@niecore

Yep, that's right! Thank you for posting here. I'll implement a better error message soon :)

**Comment by Stranger6667** [comment]
Released in `3.6.6`

Now it shows:

```
DeadlineExceeded: API response time is too slow! It took 106.86ms, which exceeds the deadline of 20.00ms.

Consider extending the deadline with the `--hypothesis-deadline` CLI option.
You can disable it completely with `--hypothesis-deadline=None`.
```
